Firestore/core/test/unit/local/target_cache_test.cc

```diff
@@ -206,6 +228,43 @@ TEST_P(TargetCacheTest, RemoveQueryRemovesMatchingKeysToo) {
   });
 }
 
+TEST_P(TargetCacheTest, RemoveTargets) {
+  persistence_->Run("test_remove_targets", [&]() {
+    TargetData target_data1 = MakeTargetData(testutil::Query("a"));
+    cache_->AddTarget(target_data1);
+    TargetData target_data2 = MakeTargetData(testutil::Query("b"));
+    cache_->AddTarget(target_data2);
+
+    std::unordered_map<model::TargetId, TargetData> live_targets;
+    cache_->RemoveTargets(target_data2.sequence_number(), live_targets);
```

Comment:
Optional: you could use empty braces to simply create the empty map on the fly:
```cpp
cache_->RemoveTargets(target_data2.sequence_number(), {});
```

Applies below as well.

Reply:
Done.